app/web/features/dashboard/DashboardUserProfileSummary.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, CircularProgress, Theme, useMediaQuery } from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import Button from "components/Button";
+import { ProfileUserProvider } from "features/profile/hooks/useProfileUser";
+import UserOverview from "features/profile/view/UserOverview";
+import useCurrentUser from "features/userQueries/useCurrentUser";
+import { DASHBOARD } from "i18n/namespaces";
+import Link from "next/link";
+import { useTranslation } from "next-i18next";
+import { routeToEditProfile, routeToProfile } from "routes";
+
+import MinimalUserProfileCard from "./MinimalUserProfileCard";
+
+function DashboardUserProfileSummaryActions() {
+  const { t } = useTranslation([DASHBOARD]);
+  return (
+    <>
+      <Link href={routeToEditProfile()} passHref>
+        <Button component="a" color="primary">
+          {t("dashboard:profile_summary_edit")}
+        </Button>
+      </Link>
+      <Link href={routeToProfile()} passHref>
+        <Button component="a" variant="outlined">
+          {t("dashboard:profile_summary_view")}
+        </Button>
+      </Link>
+    </>
+  );
+}
+
+export default function DashboardUserProfileSummary() {
+  const { data: user, error, isLoading } = useCurrentUser();
+  const desktopMode = useMediaQuery((theme: Theme) =>
+    theme.breakpoints.up("sm")
+  );
+  return (
+    <>
+      {error && <Alert severity="error">{error}</Alert>}
+      {isLoading ? (
+        <Box display="flex" justifyContent="center">
+          <CircularProgress />
+        </Box>
+      ) : user ? (
+        desktopMode ? (
```

Comment:
This could cause problems later if we want to do SSR. But I imagine we won't do that for the dashboard, so probably fine here.

Reply:
ah good catch, I hadn't considered that. When that time comes, we could simply not mount this part of the dashboard. But let me know if you prefer me to do a different implementation